generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable, Inject } from '@angular/core';
 import { Observable, Observer, Subscription } from 'rxjs/Rx';
-import { UIRouter } from 'ui-router-ng2';
+import { Router, NavigationEnd } from '@angular/router';
```

Comment:
can you just move this above? such that we will group `@angular` imports together